files/en-us/web/api/web_audio_api/advanced_techniques/index.html

```diff
@@ -493,22 +489,24 @@ <h2 id="Playing_the_audio_in_time">Playing the audio in time</h2>
     // push the note on the queue, even if we're not playing.
     notesInQueue.push({ note: beatNumber, time: time });
 
-    if (pads[0].querySelectorAll('button')[currentNote].getAttribute('aria-checked') === 'true') {
-        playSweep()
+    if (pads[0].querySelectorAll('button')[beatNumber].getAttribute('aria-checked') === 'true') {
+        playSweep(time)
     }
-    if (pads[1].querySelectorAll('button')[currentNote].getAttribute('aria-checked') === 'true') {
-        playPulse()
+    if (pads[1].querySelectorAll('button')[beatNumber].getAttribute('aria-checked') === 'true') {
+        playPulse(time)
     }
-    if (pads[2].querySelectorAll('button')[currentNote].getAttribute('aria-checked') === 'true') {
-        playNoise()
+    if (pads[2].querySelectorAll('button')[beatNumber].getAttribute('aria-checked') === 'true') {
+        playNoise(time)
     }
-    if (pads[3].querySelectorAll('button')[currentNote].getAttribute('aria-checked') === 'true') {
-        playSourceNode(audioCtx, sample);
+    if (pads[3].querySelectorAll('button')[beatNumber].getAttribute('aria-checked') === 'true') {
+        playSourceNode(audioCtx, dtmf, time);
```

Comment:
This seems to have been renamed at some point, and forgotten here, if I'm not mistaken:
```suggestion
        playSample(audioCtx, dtmf, time);
```